src/main/resources/view/MainWindow.fxml

```diff
@@ -1,23 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.net.URL?>
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.Scene?>
-<?import javafx.scene.control.Menu?>
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.image.Image?>
-<?import javafx.scene.layout.StackPane?>
-<?import javafx.scene.layout.VBox?>
-<?import javafx.scene.layout.HBox?>
+<?import java.net.*?>
```

Comment:
I think IntelliJ might have changed this to wildcard imports? Maybe we can revert this?

Reply:
alright man, i have reverted all the wildcards imports! thanks for pointing it out